MCNPInput.cpp

```diff
@@ -49,6 +49,27 @@ static void strlower( std::string& str ){
   }
 }
 
+static std::string findMnemonic( std::string token1, std::string token2 ){
```

Comment:
I recommend passing the full set of tokens into this method to avoid documentation on why you pass the 2nd and 3rd tokens only - you can then access only the ones you need and localize that documentation here.

At the same time, you might want to optionally return the `tx_id` as a pass-by-reference argument as well?